src/main.rs

```diff
@@ -166,10 +173,10 @@ fn main() -> std::result::Result<(), Box<dyn std::error::Error + 'static>> {
     }
 
     let settings = package_json.boltzmann.take().unwrap();
-    let updated_settings = settings.merge_flags("the version of boltzmann".to_string(), &flags);
+    let updated_settings = settings.merge_flags(option_env!("CARGO_PKG_VERSION").unwrap().to_string(), &flags);
```

Comment:
Oh interesting – so `option_env!` reads a value at compile time and interns it into the binary? TIL!